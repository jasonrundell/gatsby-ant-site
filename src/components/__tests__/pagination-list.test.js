import React from 'react'
import renderer from 'react-test-renderer'

import PaginationList from '../pagination-list'

describe('PaginationList', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<PaginationList path={'/path/'} pagesCount={2} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
