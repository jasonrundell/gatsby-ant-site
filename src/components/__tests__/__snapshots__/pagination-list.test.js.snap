// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationList renders correctly 1`] = `
<nav
  aria-label="Pagination Navigation"
  role="navigation"
>
  <ul
    className=""
  >
    <li
      className=""
    >
      <a
        href="/path/"
      >
        1
      </a>
    </li>
    <li
      className=""
    >
      <a
        href="/path/2"
      >
        2
      </a>
    </li>
  </ul>
</nav>
`;
